src/controllers/dlgprefcontroller.cpp

```diff
@@ -749,3 +750,11 @@ void DlgPrefController::openScript() {
         }
     }
 }
+
+
+bool DlgPrefController::presetInfoLessThan(const PresetInfo &a, const PresetInfo &b) {
+    //the compartion function for PresetInfo objects
+    //this function is used to sort the list of 
+    //presets in the combo box
```

Comment:
Please add space between // and comment. without space is for code only //code
https://www.mixxx.org/wiki/doku.php/coding_guidelines#comments